lightning-tag/detailed-reference.md

```diff
@@ -1,3 +1,96 @@
+
+```javascript
+<script>
+     var lotameTagInput = {
+         data: {},
+         config: {
+             clientId: CLIENT_ID
+         }
+     };
+
+     ! function(input) {
+         input = input || {};
+         var config = input.config || {};
+         namespace = window['lotame_' + config.clientId] = {};
+         namespace.config = config;
+         namespace.data = input.data || {};
+     } (lotameTagInput);
+ </script>
+ ```
+
+The details of the input `data` and `config` options are below.
+
+### Config object
+
+The `config` object in the Lotame Lightning Tag input has the below possible parameters:
+
+Parameter Name | Description | Type | Required? | Default
+-------------- | ----------- | ---- | :-------: | :-----:
+clientId | Your Lotame Client ID  | `int` | YES | N/A
+onProfileReady | Callback function called once Lotame targeting returns. Discussed further below in the Callback section of this page | `function(profile)` | NO | `{}`
+audienceLocalStorage | Determines whether to write audiences to the browser's `localStorage` and what the name of the key is. Discussed further below in the Local Storage section | `boolean` _**-OR-**_ `string` | NO | `false`
+
+### Data Object
+
+The `data` object is used in to pass first party-data to the Lotame DMP for use in targeting or other features of the Lotame platform. The object can be passed at setup time in the tag input function. It can also be passed post-page load in the `collect({data})` method that is described further down this page.
+
+The `data` object's parameters are fully described in [Lightning Tag Data Collection](lightning-tag/data-collection.md).
+
+## Retrieving Audiences
+
+The Lotame Lightning Tag allows you to retrieve audiences for the customer on your site. There are a few ways to do this and they are described below.
+
+### Callback
+
+As noted above in the `config` object created at script setup, there is a callback method named `onProfileReady` that can be set to be called at the completion of the targeting. This work happens asynchronously so the callback is fired when the targeting work is completed.
+
+When fired, the callback is passed a `profile` object that has the following functionality.
+
+Method Name | Return Type | Parameters | Description
+----------- | ----------- | ---------- | -----------
+`getProfileId()` | String | n/a | The id of the profile associated with the browser
+`getAudienceString( [delimiter=','], [limit])` | String | 1. `delimiter` - optional, defaults to comma. <br/> 2. `limit` - optional, defaults to no limit | Delimited string of targeting codes or audience ids, based on your established tag settings. The default, i.e. no arguments, is comma delimited with no limit. Example: `"aud1,aud2"` or `"1,2"`
```

Comment:
these should be obvious but should we call out that the parameters are character (string?) and number (integer?)?

Reply:
Makes sense. Added. Now says ... "`delimiter` - optional, string, defaults to comma." and "`limit` - optional, integer, defaults to no limit"